src/coreComponents/fileIO/logPart/unitTests/testLogPart.cpp

```diff
@@ -0,0 +1,138 @@
+             );
+  oss.clear();
+}
+
+TEST( testSection, sectionWithTitleAndOneDescription )
+{
+  std::ostringstream oss;
+  LogPart logPart( "section name" );
+  logPart.addDescription( "description name" );
+  logPart.begin( oss );
+  EXPECT_EQ( oss.str(),
+             "\n######################################################################\n"
+             "##                           section name                           ##\n"
+             "######################################################################\n"
+             "##  description name                                                ##\n\n"
+             );
+  oss.clear();
+}
+
+TEST( testSection, sectionWithSetWidth )
+{
+  std::ostringstream oss;
+  LogPart logPart( "section name" );
+  logPart.addDescription( "description name 1" );
+  logPart.addDescription( "description name 2" );
+  logPart.setMinWidth( 100 );
+  logPart.begin( oss );
+  EXPECT_EQ( oss.str(),
+             "\n####################################################################################################\n"
+             "##                                          section name                                          ##\n"
+             "####################################################################################################\n"
+             "##  description name 1                                                                            ##\n"
+             "##  description name 2                                                                            ##\n\n"
+             );
+  oss.clear();
+  oss.str( "" );
+
+  logPart.end( oss );
+  EXPECT_EQ( oss.str(),
+             "\n##                                          section name                                          ##\n"
+             "####################################################################################################\n\n"
+             );
+  oss.clear();
+}
+
+TEST( testSection, sectionMultipleDescriptions )
+{
+  std::ostringstream oss;
+  LogPart logPart( "TIMESTEP START" );
+  logPart.addDescription( "- Time :", "00h08m20s out of 2d, 21h26m40s (0% completed)", "500 s / 250000 s" );
+  logPart.addDescription( "- Delta Time :", "00h16m40s (1000 s)" );
+  logPart.addDescription( "Description test" );
+  logPart.setMinWidth( 70 );
+  logPart.begin( oss );
+  EXPECT_EQ ( oss.str(),
+              "\n######################################################################\n"
+              "##                          TIMESTEP START                          ##\n"
+              "######################################################################\n"
+              "##  - Time :       00h08m20s out of 2d, 21h26m40s (0% completed)    ##\n"
+              "##                 500 s / 250000 s                                 ##\n"
+              "##  - Delta Time : 00h16m40s (1000 s)                               ##\n"
+              "##  Description test                                                ##\n\n"
```

Comment:
shouldn't we have the following?
```suggestion
              "\n####################################################################\n"
              "##                          TIMESTEP START                        ##\n"
              "####################################################################\n"
              "##  - Time :       00h08m20s out of 2d, 21h26m40s (0% completed)  ##\n"
              "##                 500 s / 250000 s                               ##\n"
              "##  - Delta Time : 00h16m40s (1000 s)                             ##\n"
              "##  Description test                                              ##\n\n"
```

Reply:
because min width is set to 70